pillar/edx/ansible_vars/residential.sls

```diff
@@ -141,6 +141,8 @@ edx:
         - name: raven
         - name: git+https://github.com/raccoongang/xblock-pdf.git@8d63047c53bc8fdd84fa7b0ec577bb0a729c215f#egg=xblock-pdf
           extra_args: -e
+        - name: git+https://github.com/mitodl/edx-proctoring@f016be8c44e7ee8bf1bfa5fcc982b0f9d837f2f4#egg=edx-proctoring
```

Comment:
Do you mind double checking the commit hash as I clone the repo and didn't find it.

Reply:
This is the current head of the mitx/ironwood branch in the repository